webapp/channels/src/components/channel_info_rhs/menu.tsx

```diff
@@ -48,11 +63,25 @@ interface MenuItemProps {
     onClick: () => void;
 }
 
+// handle keyboard activation by pressing Enter or Space.
+function handleKeyDown(e: KeyboardEvent<HTMLDivElement>, onClick: () => void) {
+    if (e.key === 'Enter' || e.key === ' ') {
+        e.preventDefault();
+        onClick();
+    }
+}
+
 const menuItem = ({icon, text, className, opensSubpanel, badge, onClick}: MenuItemProps) => {
     const hasRightSide = (badge !== undefined) || opensSubpanel;
 
     return (
-        <div className={className}>
+        <div
+            className={className}
+            role='menuitem'
+            tabIndex={0}
```

Comment:
If we do make this a menu though, it should probably handle focus more like a dropdown menu though where the outer element receives focus and then you navigate within the items by using the arrow keys instead of tabbing between them

Reply:
removed menu/-item roles.